siriuspy/siriuspy/devices/idff.py

```diff
@@ -33,6 +33,12 @@ class _ParamPVs:
     CORRQS1CURRENT_MON = None
     CORRQS2CURRENT_MON = None
     CORRLCHCURRENT_MON = None
+    CORRQA1CURRENT_MON = None
+    CORRQB1CURRENT_MON = None
+    CORRQC1CURRENT_MON = None
+    CORRQC2CURRENT_MON = None
```

Comment:
So you decided to go with the QA, QB and QC naming? I thought we have agreed on a different solution...

Reply:
no! the label names are now IDSearch class constants. `qd1_1`, so on... I just forgot to change these variables naming accordingly. I add a commit